app/models.py

```diff
@@ -2279,7 +2279,10 @@ def atm_must_upload_doc():
             attachedDocumentURL = self.data.get('attachedDocumentURL', [])
             if attachedDocumentURL:
                 for ad in attachedDocumentURL:
-                    if not ad.lower().rsplit('.', 1)[1] in current_app.config.get('ALLOWED_EXTENSIONS'):
+                    try:
+                        if not ad.lower().rsplit('.', 1)[1] in current_app.config.get('ALLOWED_EXTENSIONS'):
```

Comment:
Would it be better to use [os.path.splitext](https://docs.python.org/2/library/os.path.html#os.path.splitext) to get the extension?